src/util/border-style.js

```diff
@@ -0,0 +1,24 @@
+
+import assign from 'object-assign'
+
+/**
+ * Utility for extracting border style props from components
+ */
+
+const sides = ['Top', 'Right', 'Bottom', 'Left']
+
+function borderStyle (props) {
+  props = props || {}
+
+  const borderStyle = props.border ? 'solid' : 'none'
+
+  return sides.reduce((acc, side) => {
+    const hasBorder = props[`border${side}`]
+
+    return assign({}, acc, hasBorder !== undefined && {
+      [`border${side}Style`]: hasBorder ? 'solid' : 'none'
```

Comment:
Might be outside of the scope you're thinking, but it might be nice if `border` props accepted either a boolean or a number, so that a number value would set `borderWidth`.

E.g. this could produce a 2px border:
```js
<Block border={2} />
```